infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/bootstrapper/OpenShiftBootstrapper.java

```diff
@@ -114,14 +114,16 @@ protected void doBootstrapAsync(String installerWebsocketEndpoint, String output
   }
 
   private void injectBootstrapper() throws InfrastructureException {
-    LOG.debug("Creating folder for bootstrapper");
+    LOG.debug(
+        "Bootstrapping {}:{}. Creating folder for bootstrapper", runtimeIdentity, openShiftMachine);
```

Comment:
Does OpenShift machine have meaningful `toString()` to use it here?

Reply:
Good catch. `OpenShiftMachine.getName()` should be there